docs/platforms/javascript/common/configuration/integrations/launchdarkly.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: LaunchDarkly
+description: "Attaches recent LD feature flag evaluations to error event context."
```

Comment:
Probably better to avoid acronyms in docs

```suggestion
description: "Attaches recent LaunchDarkly feature flag evaluations to error event context."
```